### PR TITLE
Fix/minor things

### PR DESCRIPTION
**Pull request by Velenir**
Sorting countries by alphabet
Uniform Link to /cookies

### CODE REVIEW

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr409--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr409--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr409--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr409--dxreact.review.gnosisdev.com

File: src/utils/index.ts [hunk 1]
```diff
@@ -248,4 +248,5 @@ export const estimateGas = async (
   return (type ? cb[type] : cb)(...mainParams, { ...txParams, gas: estimatedGasLimit, gasPrice: estimatedGasPrice })
 }
 
-export const geoBlockedCitiesToString = (extraCountries?: any) => Object.values({ ...BLOCKED_COUNTRIES, ...extraCountries }).join(', ') + '.'
+export const geoBlockedCitiesToString = (extraCountries?: {[p: string]: string}) =>
+  Object.values({ ...BLOCKED_COUNTRIES, ...extraCountries }).sort().join(', ') + '.'
```
Comment (W3stside) [inline src/utils/index.ts:252]: nice nice nice

**Review by W3stside — APPROVED** [review]

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr409--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr409--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr409--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr409--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr409--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr409--dxreact.review.gnosisdev.com

**Pull request merged**